README.md

```diff
@@ -1188,6 +1188,18 @@ dependency "vpc" {
 }
 ```
 
+When `skip_outputs` is used with `mock_outputs`, mocked outputs will be returned without pulling in the outputs from remote states. This can be usefull when you disable the backend initialization (`remote_state.disable_init`) in a CI by example.
```

Comment:
```suggestion
When `skip_outputs` is used with `mock_outputs`, mocked outputs will be returned without pulling in the outputs from remote states. This can be useful when you disable the backend initialization (`remote_state.disable_init`) in CI for example.
```